Index: app/src/main/java/com/click/cn/widget/ceshi.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- app/src/main/java/com/click/cn/widget/ceshi.java	(date 1545288665000)
+++ app/src/main/java/com/click/cn/widget/ceshi.java	(date 1545290828000)
@@ -12,4 +12,16 @@
 
     //------------这是master分支-------------------------
     //饿哦是测试
+
+
+
+    //---------------测试暂存--------------
+    //这是暂存
+
+
+
+
+
+
+    //------------------打开暂存区后---------------
 }
